dev/Collections/Vector.h

```diff
@@ -754,12 +754,12 @@ struct ObservableVector :
         {
         }
 
-        winrt::IInspectable Current() const
+        [[nodiscard]] winrt::IInspectable Current() const
```

Comment:
>[[nodiscard]]  [](start = 8, length = 14)

What is the guideline for when to put [[nodiscard]] ?

Reply:
http://clang.llvm.org/extra/clang-tidy/checks/modernize-use-nodiscard.html